pages/docs/features/inngest-functions/steps-workflows/step-ai-orchestration.mdx

```diff
@@ -31,9 +31,9 @@ Using [AgentKit](/docs/agent-kit/overview) and `step.ai` allows you to:
 
 ## AgentKit: AI and agent orchestration
 
-AgentKit is a simple, standardized way to implement model calling — either as individual calls, a comoplex workflow, or agentic flows.
+AgentKit is a simple, standardized way to implement model calling — either as individual calls, a complex workflow, or agentic flows.
```

Comment:
@tonyhb I'm in favor of removing this "AgentKit: AI and agent orchestration" block as this is a `step.ai` page and AgentKit is already mentioned in the `<Callout>` just above that links to its dedicated docs section